symplyphysics/laws/electricity/circuits/quality_factor_of_empty_rectangular_resonator.py

```diff
@@ -0,0 +1,57 @@
+from sympy import (Eq, solve, S)
+from sympy.physics.units import magnetic_constant
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output, convert_to, dimensionless, angle_type)
+
+
+# Description
+## A rectangular resonator consists of metal walls and a material filling it.
+## In the case when the resonator is empty, its quality factor depends only on the losses in the metal walls of the resonator.
+## The quality factor shows the ratio of the energy stored in the resonator to the energy loss during one oscillation period.
+
+## Law is: Q = w0 * mu0 * mur * b * a * l * (a^2 + l^2) / (2 * Rs * (a^3 * (l + 2 * b) + l^3 * (a + 2 * b))), where
```

Comment:
Link to the law please.

Reply:
> Link to the law please.

https://old.study.urfu.ru/view/aid/67/1/resonators.pdf

Page 24.